src/commands/TeamsMessaging.ts

```diff
@@ -2,10 +2,13 @@ import axios from "axios"
 import yargs from "yargs"
 
 export interface TeamsMessagingOptions {
-  webhook: string
+  projectName: string
   platform: string
-  artifactUrl: string
+  buildUrl: string
+  buildNumber: number
   wasBuildSuccessful: boolean
+  artifactUrl: string
+  webhook: string
 }
 
 export class TeamsMessaging {
```

Comment:
you should rename the options, class and filename to make sure it matches the command name. same for the other cmd.